tests/Unit/NumericalAlgorithms/DiscontinuousGalerkin/Test_Mass.cpp

```diff
@@ -0,0 +1,108 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "Framework/TestingFramework.hpp"
+
+#include <cstddef>
+
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tags/TempTensor.hpp"
+#include "DataStructures/Tensor/EagerMath/Determinant.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "DataStructures/Variables.hpp"
+#include "Domain/CoordinateMaps/Affine.hpp"
+#include "Domain/CoordinateMaps/CoordinateMap.hpp"
+#include "Domain/CoordinateMaps/CoordinateMap.tpp"
+#include "Domain/CoordinateMaps/ProductMaps.hpp"
+#include "Domain/CoordinateMaps/ProductMaps.tpp"
+#include "Domain/LogicalCoordinates.hpp"
+#include "NumericalAlgorithms/DiscontinuousGalerkin/Mass.hpp"
+#include "NumericalAlgorithms/Spectral/Mesh.hpp"
+#include "NumericalAlgorithms/Spectral/Spectral.hpp"
+#include "Utilities/Gsl.hpp"
+#include "Utilities/TMPL.hpp"
+
+namespace dg {
+namespace {
+
+template <size_t Dim, typename... CoordMaps>
+void test_mass(const Mesh<Dim>& mesh,
+               const domain::CoordinateMap<Frame::Logical, Frame::Inertial,
+                                           CoordMaps...>& coordinate_map,
+               const DataVector& scalar_field,
+               const DataVector& expected_massive_scalar_field) noexcept {
```

Comment:
a `massive_scalar_field` is a thing that is not this thing as far as my understanding goes. How about `expected_mass_matrix_times_scalar_field`?